### ISSUE TITLE
Feedback from SME meeting

### ISSUE THREAD

**Issue by karafecho** [NCATSTranslator/Feedback#849]
Posting feedback from meeting with Dave Peden (SME), Kara, Gwen, and Skye on July 8, 2024. This issue will need to be unpacked, but I’m posting the feedback just to capture Dave’s comments while they are fresh in my mind and scrap-paper notes.

**What drugs may treat asthma?** PK=86545425-9a66-4f52-ad5d-15278005018e [Note that this PK is from a rerun and returns somewhat different results than the results that Dave reviewed; my laptop battery died during the call, and apparently, I lost the screenshot intended to capture the PK. Sigh.]

1. The system is “glacially slow”. He’s not sure what the ideal waiting time is for returning query results, however, he noted that "it depends on how desperate I am for answers”. Maybe remove the requirement to remain on page while answers are being generated? FWIW, I find this to be very annoying.
2. Why is an experimental compound (Gsk-679586a2) returned as the top answer?
3. Descriptions of drugs/chemicals:
- Descriptions are sometimes, but not always, available; moreover, they are sometimes, but not always, available only after clicking on the drug/chemical to expand the answer. Examples of these scenarios include linolenic acid, itraconazole, pranlukast, fluticasone, salmeterol.
- Descriptions are too wordy. Dave would like to see a concise description of the chemical class and (for drugs) mechanism of action.
- Example is terbutaline:
![image](https://github.com/NCATSTranslator/Feedback/assets/26254388/b2521fe4-8117-42ad-8934-5f0fb891948f)
4. Generic results are not helpful. Example is sodium chloride.
5. Scoring:
- Description of the scoring algorithm is not clear.
- Scoring/ranking does not seem accurate or intuitive. Why is methylprednisone ranked lower than r-linolenic acid? r-linolenic acid is not a drug. Why is influenza virus vaccine ranked so high?
6. Supporting graph shows an inferred “in clinical trials for” edge, but the KL/AT points only to the ChEMBL wiki page, not the clinical trials. Example is valategrast:
![image](https://github.com/NCATSTranslator/Feedback/assets/26254388/59171cd0-8abc-4630-b4a8-0cad3e9c2518)
- Dave found many other examples where the ChEMBL wiki page was the supporting evidence in the absence of the clinical trial.
- Dave felt strongly about replacing ChEMBL clinical trials data with clinicaltrials.gov Dave. (More support for prioritizing Gwen’s effort and the CQS!)
8. ChEBI role classification categories are not helpful and seem incomplete.
For example, terbutaline is a beta agonist, but that’s not highlighted as one of the chemical role categories. 
![image](https://github.com/NCATSTranslator/Feedback/assets/26254388/dea48da3-cb01-46f4-82db-e4d1c4d45e91)
- Chemical class would be more helpful than the current ChEBI role classification categories. 
- Drug approval status is more helpful than the ChEBI role classification categories.
- "But where is the approved indication?”
9.  Wiki pages are not terribly helpful without edge-specific interpretations. Example records (as opposed to edges) were especially unhelpful.
10. Grouping by chemical class or pathway would be helpful.
11. Description and visualization of lookup versus inferred required explanation, as the current definition was not only insufficient by not easy for Dave to find and the visualization was not intuitive.

**What chemicals may downregulate MYD88?**

We ran out of time before being able to take a deep dive into the result set, but Dave seemed more satisfied with these results than the “may treat” results, in agreement with Andy's observation that we set the bar too high for MVP1.

**Comment by gprice1129** [comment]
Adding the label just to communicate that the UI team plans on looking into this feedback which may alter our plans for Fugu.

**Comment by gaurav** [comment]
NodeNorm [has a shorter description for terbutaline](https://nodenorm.transltr.io/get_normalized_nodes?curie=CHEBI%3A9449&conflate=true&drug_chemical_conflate=false&description=true) ("A member of the class of phenylethanolamines that is catechol substituted at position 5 by a 2-(tert-butylamino)-1-hydroxyethyl group."), but I personally prefer the description we currently use (provided by the Annotator, I think?).

**Comment by sharatisrani** [comment]
Comment 2: GSK679.... have entered comment to [issue #840](https://github.com/NCATSTranslator/Feedback/issues/840), tying this issue there - Improving is working on a fix.

Comment 5B: Things have changed.  r-linolenic acid is no longer returned.  Three linolenic acids are returned, and the highest is at #79, while methylprednisone is at #46.   Influenza virus vaccine is at #587, unlikely to see the light of day.

Comment 10: These features are underway, hopefully in Guppy.  Very useful to get this endorsement.

**Comment by sharatisrani** [comment]
Comment 8: Are we not switching to ATC Level 4?   @Genomewide @jh111

**Comment by sharatisrani** [comment]
Comments 4, 6: it will be interesting to see how these are solved.  These will have an overall improvement in result quality, and similar comments have been raised in other issues.

**Comment by khanspers** [comment]
"Additionally, allowing users to browse results before they are complete is probably not helpful and leads to more confusion (and weird results) than the quick intermediary results are worth." -> I agree with this, it is kind of confusing and not intuitive.
If the purpose of showing intermediate results is to keep the user engaged and show that something is indeed being returned, maybe the intermediate results should be greyed out and not clickable, to indicate that they are not complete? If the purpose is specifically to allow and encourage users to browse/view incomplete results, then maybe the status message at the top that tells you there are new results should be more prominent (color, size...).

**Comment by sstemann** [comment]
> Adding the label just to communicate that the UI team plans on looking into this feedback which may alter our plans for Fugu.

@gprice1129 are there UI changes expected for Fugu for this? otherwise, lets remove the label

**Comment by sierra-moxon** [comment]
agree with @khanspers on "load new results" -- @Genomewide also mentioned in a call yesterday, that the slow return of text as tools like chatgpt are running signals the user that the system is still thinking...

**Comment by sstemann** [comment]
> Comment 8: Are we not switching to ATC Level 4? @Genomewide @jh111

based on latest NCATS/UI call we are not switching to ATC Level 4 in the UI without additional user testing.  This is pending.
